client/src/components/layout/MyTablesList.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react"
+import TableTile from "./TableTile.js"
+
+const MyTablesList = (props) => {
+  const [tables, setTables] = useState([])
+
+  const getTables = async () => {
+    try {
+      const response = await fetch("/api/v1/tables/currentUser")
```

Comment:
this route is probably more like a `/api/v1/users/:id/tables` for all the tables for the current user